Extensions/ParticleSystem/particleemitterobject-pixi-renderer.ts

```diff
@@ -119,7 +119,7 @@ namespace gdjs {
               },
               minStart: objectData.emitterForceMax,
```

Comment:
btw, should this be `emitterForceMin` ?
We don't use this prop anymore

Reply:
Indeed, it's just that the whole configuration values are overridden by the setters at the start so it didn't do a bug.

Reply:
nice